test_utils/renderRouteWithStore.js

```diff
@@ -20,8 +20,11 @@ export function renderRouteWithStore(
   initialRoute = '/',
   container,
   state = initialState(),
-  history = createMemoryHistory()
+  storage = {},
```

Comment:
What if we set the user object here as the default and use and empty object in login tests?
We will be using the authenticated version most of the time.

Reply:
Yeah that makes sense.

I think actually we can drop also `initialState`.

I see some refactoring to do. I think better in a new PR.